tools/tsp-client/test/commands.spec.ts

```diff
@@ -286,6 +324,27 @@ describe.sequential("Verify commands", () => {
     }
   }, 360000);
 
+  it("Generate config files with alternate json path", async () => {
```

Comment:
Same for this test, should we try deleting those files and then asserting that they exist?